storage/dataflux/next_page.go

```diff
@@ -0,0 +1,184 @@
+// Copyright 2024 Google LLC
```

Comment:
Can you indicate in the file name that this is next page only for worksteal listing (`worksteal_next_page` maybe)?